scripts/ci/reborn_pr_test_plan.py

```
@@ -823,6 +834,24 @@ def build_plan(
+            package = next(
+                (
+                    name
+                    for directory, name in package_directories.items()
+                    if owner.startswith(f"{directory}/")
+                ),
+                None,
+            )
+            if package is None:
+                raise ValueError(
+                    f"repository config owner is in no workspace package: {owner}"
+                )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)